Shoko.Server/Server/ShokoEventHandler.cs

```diff
@@ -97,5 +98,13 @@ internal void OnEpisodeUpdated(DataSourceEnum source, SVR_AniDB_Anime anime, SVR
                 EpisodeInfo = episode,
             });
         }
+
+        internal void OnSettingsSaved()
+        {
+            SettingsSaved?.Invoke(null, new SettingsSavedEventArgs
+            {
+                
+            });
```

Comment:
```suggestion
            SettingsSaved?.Invoke(null, new SettingsSavedEventArgs());
```

no point having the extra lines/object constructor if nothing is used, we can always add it in if needed.